mopidy_spotify/search.py

```diff
@@ -55,11 +55,18 @@ def search(config, session, requests_session,
             'to at most 50.')
         search_count = 50
 
+    params = {
+        'q': sp_query,
+        'limit': search_count,
+        'type': ','.join(types)}
+
+    try:
+        params['market'] = session.user_country
+    except ValueError as exc:
```

Comment:
Hm, in libspotify it is an int and pyspotify doesn't raise any errors no matter what the value is, so I don't see how anything could happen. If the int is 0, you will get null-bytes, but that shouldn't trigger a ValueError.